packages/cli/src/services/test-filters.ts

```diff
@@ -3,9 +3,9 @@ export function filterByCheckNamePattern (checkNamePattern = '', checkName: stri
   return re.test(checkName)
 }
 
-export function filterByFileNamePattern (filePatterns: Array<string> = [], path: string) {
+export function filterByFileNamePattern (filePatterns: Array<string> = [], path: string|undefined) {
```

Comment:
`path` was actually `string|undefined`before (it was called with `scriptPath`, which is null for API checks). Typechecking was still passing, though, since we were using `any` to represent checks.

`re.test(undefined)` works fine and returns `true`, so this is OK. There are no behavioral changes here - this just updates types to satisfy TS.